src/panel/applets/icon-tasklist/widgets/ButtonPopover.vala

```diff
@@ -257,6 +257,11 @@ private class WindowControls : Gtk.Box {
 			relief = Gtk.ReliefStyle.NONE,
 		};
 
+		// Keep on top is not currently supported on Wayland
+		if (libxfce4windowing.windowing_get() == libxfce4windowing.Windowing.WAYLAND) {
+			keep_on_top_button.sensitive = false;
+		}
```

Comment:
Given we aren't going to support X11, I think it'd be better just to remove the functionality entirely, as 10.10+ this button would just be permanently disabled.

Reply:
It should be possible to support always-on-top on Wayland, but it has to be implemented by libxfce4windowing. Using labwc, I can set a window to always be on top from the right click menu on a window's titlebar. I'm not sure which protocol is used for this; maybe [`wlr-layer-shell`](https://gitlab.freedesktop.org/wlroots/wlroots/-/blob/master/protocol/wlr-layer-shell-unstable-v1.xml?ref_type=heads)?